mscore/tourhandler.h

```diff
@@ -0,0 +1,103 @@
+#ifndef TOURHANDLER_H
+#define TOURHANDLER_H
+
+#include "libmscore/xml.h"
+
+namespace Ms {
+
+//---------------------------------------------------------
+//   TourMessage
+//---------------------------------------------------------
+
+struct TourMessage {
+      QString message;
+      QList<QString> widgetNames;
+      void init(QString m, QList<QString> w) { message = m; widgetNames = w; }
```

Comment:
_Disclaimer: I am definitely no pro in C++, so feel free to disagree._

From my point of view it seems more concise to use a constructor and initialization list here:
```cpp
TourMessage(QString m, QList<QString> w) : message(m), widgetNames(w) {}
```